geniza/corpus/models.py

```diff
@@ -1080,18 +1141,26 @@ def index_data(self):
                 "num_translations_i": counts[Footnote.TRANSLATION],
                 "num_discussions_i": counts[Footnote.DISCUSSION],
                 # count each unique source as one scholarship record
-                "scholarship_count_i": self.sources().count(),
+                "scholarship_count_i": len(source_relations.keys()),
                 # preliminary scholarship record indexing
                 # (may need splitting out and weighting based on type of scholarship)
-                "scholarship_t": [fn.display() for fn in self.footnotes.all()],
+                "scholarship_t": [fn.display() for fn in footnotes.all()],
```

Comment:
```suggestion
                "scholarship_t": [fn.display() for fn in footnotes],
```

Reply:
Wow, thank you for finding that! I thought that using `.all()` would just return the prefetched content, but there must be cases where it doesn't